db/migrate/20230614135144_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :admin, :boolean, :default => false
+  end
+end
```

Comment:
WDYT about adding roles instead of `admin` column?

Reply:
Hm, thanks i try implement it